plugins/postcss-design-tokens/src/data-formats/parse-import.ts

```diff
@@ -0,0 +1,67 @@
+import valueParser from 'postcss-value-parser';
+import { Token } from './base/token';
+import { extractStyleDictionaryTokens } from './style-dictionary/style-dictionary';
+import path from 'path';
+import { promises as fsp } from 'fs';
+
+function parseImport(statement: string): { filePath: string, format: string, conditions: Array<string> } {
+	const importAST = valueParser(statement);
+
+	const result = {
+		filePath: '',
+		format: 'standard',
+		conditions: ['6b4e71e7-4787-42f7-a092-8684961895db'], // a random, but shared default condition
```

Comment:
Not entirely sure what this is from reading the code, given it's shared below, should it be made a constant instead of a literal?

Reply:
Made it a constant